db/db_basic_test.cc

```diff
@@ -4361,6 +4361,39 @@ TEST_P(DBBasicTestDeadline, IteratorDeadline) {
   Close();
 }
 
+#ifndef ROCKSDB_LITE
```

Comment:
Can we move this test to db_wal_test.cc? Sorry I should have put it there in the first place.

Reply:
Yes, I will move it to in db_wal_test.cc